File: src/controle-fluxo-empresarial/src/models/Pessoas/Pessoa.ts

import { BaseEntity } from './../BaseEntity';
export interface Pessoa extends BaseEntity {
    apelido?: string | null,
    bairro?: string | null,
    cep?: string | null,
    cidadeId?: number | null;
    complemento?: string | null,
    cpfcpnj?: string | null,
    email?: string | null,
    endereco?: string | null,
    nome?: string | null,
    numero?: string | null,
    observacao?: string | null,
    rgInscricaoEstadual?: string | null,
    telefone?: string | null,
}

export enum SEXO {
    Masculino = "Masculino",
    Feminino = "Feminino",
    Outros = "Outros",
}

export enum ESTADO_CIVIL {
    Casado = "Casado",
    Divorciado = "Divorciado",
    Separadoo = "Separadoo",
    Solteiro = "Solteiro",
    Viuvo = "Viuvo",
    Outros = "Outros",
}

export enum TIPO_PESSOA {
    Fisica = "Fisica",
    Juridica = "Juridica",
}

File: src/ControleFluxoEmpresarial/Models/Pessoas/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ControleFluxoEmpresarial.Models.Pessoas
{
    public class Pessoa : BaseModelSituacao
    {
        public int CidadeId { get; set; }

        public string Apelido { get; set; }

        public string Bairro { get; set; }

        public string Cep { get; set; }

        public string Complemento { get; set; }

        public string CPFCPNJ { get; set; }

        public string Email { get; set; }

        public string Endereco { get; set; }

        public string Nome { get; set; }

        public string Numero { get; set; }

        public string Observacao { get; set; }

        public string RgInscricaoEstadual { get; set; }

        public string Telefone { get; set; }
    }
}
